api/src/main/java/org/apache/iceberg/actions/RewritePositionDeleteFiles.java

```diff
@@ -41,11 +119,60 @@
   RewritePositionDeleteFiles filter(Expression expression);
 
   /** The action result that contains a summary of the execution. */
+  @Value.Immutable
   interface Result {
+    List<PositionDeleteGroupRewriteResult> rewriteResults();
+
     /** Returns the count of the position deletes that been rewritten. */
```

Comment:
The doc in all these 4 methods must use `position delete files` rather than `position deletes` since these methods represent file counts.